package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "build-staging": "vite build  --base=/app/ --mode preprod",
-    "build-prod": "vite build --base=/app/ --mode prod",
+    "build-staging": "vite build mode preprod",
```

Comment:
the CI issue is probably because you changed "--mode" to "mode"

Reply:
fix here : https://github.com/openfoodfacts/open-prices-frontend/commit/011d5a1e25419824670fc1956a7098225058a064

Reply:
wow sorry for that :sweat_smile: